### ISSUE TITLE
DatePicker onBlur Method Triggered Twice, Returning undefined on Second Call, when Selecting date from Picker

### ISSUE THREAD

**Issue by devcaco** [ant-design/ant-design#51841]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/antd-reproduction-template-forked-rjd2tv)

### Steps to reproduce

Assign a onBlur method to the DatePicker and console log the value. When selecting date from picker the onBlur method gets fired twice with the second call returns undefined value.

### What is expected?

For the onBlur method to get called only once with the correct value when selecting date from the picker

### What is actually happening?

When selecting a date from the DatePicker, the onBlur method is triggered twice: the first call provides the correct value, while the second call returns undefined.

| Environment | Info |
| --- | --- |
| antd | 5.22.2 |
| React | React 18.3.1 |
| System | MacOS |
| Browser | Chrome 131.0.6778.86 |

**Comment by devcaco** [comment]
It appears that for some reason the second call is from an element inside the date picker. - If I console log the event's target name or id its different from the first call.

**Comment by MadCcc** [comment]
duplicate of https://github.com/ant-design/ant-design/issues/51540
